### DISCUSSION TITLE
Дополнительный функционал субтитров

### DISCUSSION THREAD

**Discussion by Neovolter** [ilyhalight/voice-over-translation#756] [Ideas]
Огромное спасибо за отличное расширение! :) Было бы здорово иметь функционал субтитров как в Яндекс Браузере (интеграция Яндекс Переводчика - когда при нажатии на слово в тексте субтитров высвечивается его перевод и перевод всего предложения)
![image](https://github.com/user-attachments/assets/c90d82ee-6494-4448-80aa-f0ffac175d05)

**Comment by ilyhalight** [comment]
Не скажу насколько скоро, но будет добавлено, по предложениям в дальнейшем пишите в Issues
